spec/index.html

```diff
@@ -696,7 +696,8 @@ <h2>Literals</h2>
     <ul>
       <li>a <dfn>lexical form</dfn> consisting of a sequence of
         <a data-lt="code point" class="lint-ignore">Unicode code points</a> [[!UNICODE]]
-        matching the <a data-cite="XML10#charsets">Char</a> production [[XML10]].</li>
+        matching the <a data-cite="XML11#charsets">Char</a> production [[XML11]].
+       Because RDF lexical forms match the Char production of XML they do not contain Unicode surrogate code points.</li>
```

Comment:
```suggestion
        matching the <a data-cite="XML11#charsets">Char</a> production of [[XML11]].
        Since RDF lexical forms match the <a data-cite="XML11#charsets">Char</a>
        production of [[XML11]], they do not contain Unicode surrogate code points.</li>
```

Reply:
If we remove the restriction for the Char production, literals may contain surrogates.

Reply:
It would be possible to limit the "lexical form" of language-tagged strings - to scalar values, for example -  but make the space of lexical forms larger - code points or even 32-bit integers.

Reply:
```suggestion
        which are <a data-cite="i18n-glossary#dfn-scalar-value">Unicode scalar values</a>, thus, they do not contain Unicode surrogate code points.</li>
```